content/legal/privacy/_index.md

```diff
@@ -85,13 +85,13 @@ Information we collect:
 
 * Name
 * Email
-* Username
-* Phone Number (optional / required if using 2-factor authentication)
```

Comment:
Not sure if people really do phone based 2fa anymore? With SIM swapping attacks, I think it's discouraged...